src/drawing/DrawingSVG.tsx

```diff
@@ -18,7 +18,7 @@ export function DrawingSVG({ className, children, ...rest }: Props, ref: React.R
             {...rest}
         >
             <defs>
-                <filter filterUnits="userSpaceOnUse" height="100vh" id="ba-DrawingSVG-shadow" width="100vw">
+                <filter filterUnits="userSpaceOnUse" id="ba-DrawingSVG-shadow">
```

Comment:
I thought this was what we had previously?

Reply:
We previously had `primitiveUnits="objectBoundingBox"` in the first commit for this file. It was later changed to `filterUnits`.